public/config/configuration.json

```diff
@@ -6,5 +6,8 @@
   "PAY_API_URL": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1/",
   "KEYCLOAK_CONFIG_URL": "/local-keycloak-config-url/keycloak.json",
   "ADDRESS_COMPLETE_KEY": "RH99-RN99-GB72-FW86",
-  "LD_CLIENT_ID": "5e5d5af4574f7e09ab53d03b"
```

Comment:
Please restore this key that you removed.

Reply:
This one actually is the original key - git is picking it up because of the comma at the end which was added. ;)